docs/source/external-tools/cdf/cdf_requirements.rst

```diff
@@ -208,8 +210,20 @@ This field allows software to properly format the associated data when displayed
 
 ``LABLAXIS``
 ^^^^^^^^^^^^
+Required if not using LABL_PTR_i.  
+
 Used to label a plot axis or to provide a heading for a data listing. This field is generally 6-10 characters.
 
+``LABL_PTR_i``
+^^^^^^^^^^^^^^
+Required if not using LABLAXIS.
+
+Used to label a dimensional variable when one value of LABLAXIS is not sufficient to describe the variable or to label all the axes. LABL_PTR_i is used instead of LABLAXIS, where i can take on any value from 1 to n where n is the total number of dimensions of the original variable. 
```

Comment:
```suggestion
Used to label a dimensional variable when one value of ``LABLAXIS`` is not sufficient to describe the variable or to label all the axes. ``LABL_PTR_i`` is used instead of ``LABLAXIS``, where i can take on any value from 1 to n where n is the total number of dimensions of the original variable. 
```